assets/js/googlesitekit/datastore/user/key-metrics.js

```diff
@@ -476,8 +478,14 @@ const baseSelectors = {
 						return false;
 					}
 
-					if ( widget.displayInList ) {
-						return widget.displayInList( select, isViewOnly );
+					if (
+						// The call to displayInList() was added solely for metrics
+						// with custom dimensions, so adding this check till we refactor this
+						// whole isViewOnly section.
```

Comment:
Do we have an issue for that?

Reply:
Yes, #9548 removed this completely and moves it to getUserPickedMetrics via a new callback.